docker/init.sh

```diff
@@ -1,4 +1,3 @@
-yarn install --check-files
```

Comment:
I wonder if we still need this here. Because if you rerun the machine after it's built, but the package.json has changed upstream on git, this will need to run else the things below will fail.

Reply:
We don't. If you pass `--build`, it will automatically rebuild if the `package.json` file changes.